validation/src/test/scala/hmda/validation/rules/lar/quality/Q024Spec.scala

```diff
@@ -0,0 +1,61 @@
+package hmda.validation.rules.lar.quality
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class Q024Spec extends LarEditCheckSpec {
+  val config = ConfigFactory.load()
+  val min_income = config.getInt("hmda.validation.quality.Q024.min_income_for_high_loan")
```

Comment:
Should this variable be camel case instead of snake case?  I understand it's referencing a variable with a different naming convention, but (I think) variables need to match the pattern in the rest of the file.